lib/graphite_async.ml

```diff
@@ -8,7 +8,7 @@ let timer_ms () =
   let start_ts = Time.now () in
   fun () ->
     let end_ts = Time.now () in
-    Time.diff end_ts start_ts |> Time.Span.to_ms |> int_of_float
```

Comment:
Perhaps it would make sense to move this conversion directly into `add_percentile_observation_opt` (instead of replicating it several places below).

Reply:
`add_percentile_observation_opt` can be used for any kind of observation you want to get percentiles of, it is not limited to time spans.